ci/test-locally.sh

```diff
@@ -24,9 +23,25 @@ cleanup() {
     set -x
 }
 trap cleanup EXIT
-
 trap "exit 24" INT TERM
 
+python3 -m pip install --user -U ../batch
+
+if [[ $CLOUD_SQL_PROXY -eq 1 ]]; then
+    export CLOUD_SQL_CONFIG_PATH=`pwd`/batch-secrets/batch-test-cloud-sql-config.json
+    connection_name=$(jq -r '.connection_name' $CLOUD_SQL_CONFIG_PATH)
+    port=$(jq -r '.port' $CLOUD_SQL_CONFIG_PATH)
+    ./cloud_sql_proxy -instances=$connection_name=tcp:$port &
+    proxy_pid=$!
```

Comment:
Also, I have a PR that would need to add to here the following to prevent race conditions:

```bash
if [[ $IN_CLUSTER -eq 0 ]]; then
    export CLOUD_SQL_CONFIG_PATH=`pwd`/batch-secrets/batch-test-cloud-sql-config.json
    connection_name=$(jq -r '.connection_name' $CLOUD_SQL_CONFIG_PATH)
    host=$(jq -r '.host' $CLOUD_SQL_CONFIG_PATH) # Add this line
    port=$(jq -r '.port' $CLOUD_SQL_CONFIG_PATH)
    ./cloud_sql_proxy -instances=$connection_name=tcp:$port &
    proxy_pid=$!
    ../until-with-fuel 30 curl -fL $host:$port # Add this line
else
    export CLOUD_SQL_CONFIG_PATH=/batch-secrets/batch-test-cloud-sql-config.json
fi
```